src/core/server/elasticsearch/client/configure_client.ts

```diff
@@ -67,15 +67,15 @@ function getResponseMessage(event: RequestEvent): string {
   return `${event.statusCode}\n${params.method} ${url}${body}`;
 }
 
-const addLogging = (client: Client, logger: Logger, logQueries: boolean) => {
+const addLogging = (client: Client, logger: Logger) => {
+  const queryLogger = logger.get('query');
```

Comment:
NIT: I would have used `addLogging(client, logger).get('query')` instead of creating the sub-logger in `addLogging `, but one could argue that we could need multiple distinct loggers in `addLogging`, so I guess this is just fine.

Reply:
Yeah, I was considering the future needs here for additional logging. Happy to change if you'd like though, don't feel too strongly about it.

Reply:
+1 to minimize the number of params. Should we add a logger with another context, we can come back to the current signature.

Reply:
Done in 5f17c38